helloworld/contracts/assembly/contracts/helloworld.ts

```diff
@@ -16,3 +18,13 @@ export function constructor(_: StaticArray<u8>): void {
   // In this case, we're emitting an event with the message "Hello, World!".
   generateEvent('Hello, World!');
 }
+
+/**
+ * @param _ - not used
+ * @returns the emitted event serialized in bytes
+ */
+export function event(_: StaticArray<u8>): StaticArray<u8> {
+  const message = 'Hello, World!';
+  generateEvent(message);
+  return stringToBytes(message);
+}
```

Comment:
That a goood idea but I think we should keep the hello world simple. Maybe you can add a param to the constructor to set the message.